python/core/fundamental-data-structures/the-fundamental-tuples.md

```diff
@@ -2,97 +2,96 @@
 author: catalin
 
 tags:
-
   - introduction
 
-levels:
-
-  - basic
-
 type: normal
 
 category: must-know
+
 aspects:
   - introduction
 
-standards:
-  python.native-types-operations.4: 10
-
 links:
-
-  - '[docs.python.org](https://docs.python.org/3.5/tutorial/datastructures.html#tuples-and-sequences){website}'
-
+  - '[Tuples and Sequences](https://docs.python.org/3.5/tutorial/datastructures.html#tuples-and-sequences){website}'
 
 ---
 
-# The fundamental `tuple`
+# The Fundamental `tuple`
 
 ---
 ## Content
 
 `Tuples` are another one of the three currently implemented **sequence data types** (alongside with `lists` and `ranges`).
```

Comment:
implemented is wrong here - let's use something like "covered so far" ? 

or even better let's not imply the sequence of insights. we can say something like:

`Tuples` are one of the fundamental sequence data types, alongside `lists` and `ranges`